packages/kbn-coloring/src/shared_components/color_mapping/color/color_handling.ts

```diff
@@ -37,31 +40,32 @@ export function getAssignmentColor(
 }
 
 export function getColor(
-  color: ColorMapping.ColorCode | ColorMapping.CategoricalColor,
+  color:
+    | ColorMapping.ColorCode
+    | ColorMapping.CategoricalColor
+    | (ColorMapping.LoopColor & { paletteId: string; colorIndex: number }),
   getPaletteFn: ReturnType<typeof getPalette>,
   isDarkMode: boolean
-) {
+): string {
   return color.type === 'colorCode'
     ? color.colorCode
-    : getValidColor(getPaletteFn(color.paletteId).getColor(color.colorIndex, isDarkMode)).hex();
+    : color.type === 'loop'
+    ? getValidColor(
+        getPaletteFn(color.paletteId).getColor(color.colorIndex, isDarkMode, true)
+      ).hex()
+    : getValidColor(
+        getPaletteFn(color.paletteId).getColor(color.colorIndex, isDarkMode, true)
+      ).hex();
```

Comment:
Aren't both branches the same line?

Reply:
correct nice spot: fixed here https://github.com/elastic/kibana/pull/175144/commits/5d94f2fb681383a887a0870092e4308420bff75c